### PR TITLE
add copy task id feature for all tables

### PR DESCRIPTION
**Pull request by lambdagirl**

### CODE REVIEW

File: ui/src/components/CompletedTasksTable.tsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ import Alert from "@material-ui/lab/Alert";
 import AlertTitle from "@material-ui/lab/AlertTitle";
 import SyntaxHighlighter from "./SyntaxHighlighter";
 import { AppState } from "../store";
+import FileCopyOutlinedIcon from '@material-ui/icons/FileCopyOutlined';
```
Comment (hibiken) [inline ui/src/components/CompletedTasksTable.tsx:23]: nit: Can you place this import so that it's under other `@material-ui/xxx` imports?

File: ui/src/components/PendingTasksTable.tsx [hunk 2]
```diff
@@ -1,45 +1,36 @@
...
-import { TableColumn } from "../types/table";
-import { TaskInfoExtended } from "../reducers/tasksReducer";
 import { taskDetailsPath } from "../paths";
+import { TaskInfoExtended } from "../reducers/tasksReducer";
+import { AppState } from "../store";
+import { TableColumn } from "../types/table";
+import { prettifyPayload, uuidPrefix } from "../utils";
+import SyntaxHighlighter from "./SyntaxHighlighter";
+import TableActions from "./TableActions";
+import TablePaginationActions, { rowsPerPageOptions } from "./TablePaginationActions";
```
Comment (hibiken) [inline ui/src/components/PendingTasksTable.tsx:33]: nit: Undo this change? I don't think it's related to the change.

File: ui/src/components/PendingTasksTable.tsx [hunk 3]
```diff
@@ -345,8 +342,20 @@ function Row(props: RowProps) {
           />
         </IconButton>
       </TableCell>
-      <TableCell component="th" scope="row">
+      <TableCell component="th" scope="row" style={{width:'200px'}}>
```
Comment (hibiken) [inline ui/src/components/PendingTasksTable.tsx:345]: nit: Avoid inline styles if possible. Add className and specify the styles like others.

Also, don't we need to apply this style in other tables too?

**Review by hibiken — COMMENTED** [review]
Thank you for opening this PR!

Left a few comments, please let me know if you have any questions :)

File: ui/src/components/PendingTasksTable.tsx (hunk 2 above)
Comment (lambdagirl) [reply ui/src/components/PendingTasksTable.tsx:33]: Imports should be sorted alphabetically.

File: ui/src/components/PendingTasksTable.tsx [hunk 4]
```diff
@@ -95,7 +86,6 @@ function PendingTasksTable(props: Props & ReduxProps) {
   const [page, setPage] = useState(0);
   const [selectedIds, setSelectedIds] = useState<string[]>([]);
   const [activeTaskId, setActiveTaskId] = useState<string>("");
-
```
[the remark this answers is not in the document]
Comment (lambdagirl) [reply ui/src/components/PendingTasksTable.tsx:98]: done!

File: ui/src/components/PendingTasksTable.tsx (hunk 3 above)
Comment (lambdagirl) [reply ui/src/components/PendingTasksTable.tsx]: done!

File: ui/src/components/CompletedTasksTable.tsx (hunk 1 above)
Comment (lambdagirl) [reply ui/src/components/CompletedTasksTable.tsx]: done!

File: ui/src/components/ScheduledTasksTable.tsx [hunk 5]
```diff
@@ -374,8 +388,23 @@ function Row(props: RowProps) {
...
+          <IconButton
+            onClick={(e) => {
+              e.stopPropagation()
+              navigator.clipboard.writeText(task.id)
+            }
+            }
+            size="small"
+            className={classes.copyButton}
+          >
+            <FileCopyOutlinedIcon fontSize="small" style={{ height: "12px", width: "12px" }} />
```
Comment (hibiken) [inline ui/src/components/ScheduledTasksTable.tsx:404]: remove this inline style

File: ui/src/components/ActiveTasksTable.tsx [hunk 6]
```diff
@@ -304,8 +311,23 @@ function Row(props: RowProps) {
...
+          <IconButton
+            onClick={(e) => {
+              e.stopPropagation()
+              navigator.clipboard.writeText(task.id)
+            }
+            }
+            size="small"
+            className={classes.copyButton}
+          >
+            <FileCopyOutlinedIcon fontSize="small" style={{ height: "12px", width: "12px" }} />
```
Comment (hibiken) [inline ui/src/components/ActiveTasksTable.tsx:327]: remove this inline style

File: ui/src/components/ArchivedTasksTable.tsx [hunk 7]
```diff
@@ -343,8 +350,23 @@ function Row(props: RowProps) {
...
+          <IconButton
+            onClick={(e) => {
+              e.stopPropagation()
+              navigator.clipboard.writeText(task.id)
+            }
+            }
+            size="small"
+            className={classes.copyButton}
+          >
+            <FileCopyOutlinedIcon fontSize="small" style={{ height: "12px", width: "12px" }} />
```
Comment (hibiken) [inline ui/src/components/ArchivedTasksTable.tsx:366]: remove inline style

File: ui/src/components/CompletedTasksTable.tsx [hunk 8]
```diff
@@ -318,8 +319,23 @@ function Row(props: RowProps) {
...
+          <IconButton
+            onClick={(e) => {
+              e.stopPropagation()
+              navigator.clipboard.writeText(task.id)
+            }
+            }
+            size="small"
+            className={classes.copyButton}
+          >
+            <FileCopyOutlinedIcon fontSize="small" style={{ height: "12px", width: "12px" }} />
```
Comment (hibiken) [inline ui/src/components/CompletedTasksTable.tsx:335]: remove inline style

File: ui/src/components/RetryTasksTable.tsx [hunk 9]
```diff
@@ -378,8 +379,23 @@ function Row(props: RowProps) {
...
+          <IconButton
+            onClick={(e) => {
+              e.stopPropagation()
+              navigator.clipboard.writeText(task.id)
+            }
+            }
+            size="small"
+            className={classes.copyButton}
+          >
+            <FileCopyOutlinedIcon fontSize="small" style={{ height: "12px", width: "12px" }} />
```
Comment (hibiken) [inline ui/src/components/RetryTasksTable.tsx:395]: remove inline style
